insolar/gen/reference.go

```diff
@@ -28,6 +28,11 @@ func ID() (id insolar.ID) {
 	return
 }
 
+func IDPointer() *insolar.ID {
```

Comment:
There is no point to making a function for that.

Reply:
Agree. Fixed.